pokemon-minisite/src/app/style/variables.scss

```diff
@@ -44,3 +44,28 @@
 
     /* FONTS */
     $main-font: 'apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Open Sans', 'Helvetica Neue', sans-serif;
+
+    @mixin btn-type-button {
```

Comment:
add the mixin to another file - mixim.scss in the style folder

Reply:
done